### ISSUE TITLE
Roadmap?

### ISSUE THREAD

**Issue by innat** [ayyucedemirbas/KerasFuse#16]
- What is the roadmap?
- What would be the difference between [keras_cv](https://github.com/keras-team/keras-cv)? Because
    - Keras-cv has the plan to add UNet (model)
    - Keras-cv has the plan to add Dice / IOU (loss/metric)
    - Keras-cv has the plan to add a 3D model as well.
    - Keras-cv has the plan to add domain-specific [data augmentation](https://github.com/keras-team/keras-cv/issues/163), slowly but surely.

FYI, in favor of Keras-cv and keras-nlp, the development of TensorFlow-addons is no longer proceeding. https://www.tensorflow.org/addons

I think it would be better to contribute directly to KerasCV.

**Comment by ayyucedemirbas** [comment]
Hello,

Our roadmap includes several exciting features and improvements. We are currently working on providing blocks and layers specifically designed for medical datasets and pretrained models on medical images, as well as developing preprocessing techniques tailored to medical images. These additions will enhance the capabilities of our library in the medical domain. I work on AI for medical images and I know what I'm doing. You might be aware that torchvision exists but that does not mean MONAI is redundant.

I doubt your sincerity and motives. You seem to be attacking our project and undermining us on GitHub and Twitter. If you want to contribute to Keras-CV, go ahead and do that. Please stop bothering us.
 
Thank you so much!

**Comment by innat** [comment]
@ayyucedemirbas 
I meant no disrespect. And I think you clearly missed the whole point I am trying to make. Disappointing
